### ISSUE TITLE
ILB-Arena: @angular-eslint has never produced a real measurement

### ISSUE THREAD

**Issue by ofri-peretz** [ofri-peretz/eslint#897]
Surfaced while fixing #891, and it is independent of ESLint version — it fails on v8, v9 and v10 alike.

`configs/angular.config.js` enables **every** rule the plugin exports. Some of them need a TypeScript program:

```
Error while loading rule '@angular-eslint/no-developer-preview':
You have used a rule which requires type information, but don't have
parserOptions set to generate type information for this file.
```

The arena corpus is plain `.js` with no project, so the lint throws.

## Why nobody noticed

`runEslint` used to catch a lint-run failure, print a `⚠️`, and return `[]`. An empty result is indistinguishable from a plugin that genuinely found nothing, so angular has been scored **0/40 TP, F1 0.0%** on every arena run ever published — a fabricated measurement presented alongside real ones.

Run failures are now fatal (#891 work), which is what exposed this.

## Current state

Angular is declared `unmeasurable` in the plugin table with its reason, and is reported as *not scored* rather than scored zero. That is honest but it is not a measurement.

## To actually fix

Either give the corpus a TypeScript program the plugin can use, or restrict the config to the rules that work without one. Metadata filtering does **not** work here — `requiresTypeChecking` is set on 0 of its 50 rules, so the type-aware ones cannot be identified programmatically:

```
total 50 | requiresTypeChecking 0
```

Worth checking whether any other benchmarked peer is in the same position now that the failure is loud.
